understanding/sensory-characteristics.html

```diff
@@ -58,203 +56,132 @@
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.3.3:</span>Sensory Characteristics (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.3.3:</span>感覚的な特徴 (レベル A)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Instructions provided for understanding and operating content do not rely solely on
-                     sensory characteristics of components such as shape, color, size, visual location,
-                     orientation,
-                     or sound.
+                  <p>コンテンツを理解し操作するための説明は、形、色、大きさ、視覚的な位置、方向、又は音のような、構成要素が持つ感覚的な特徴だけに依存していない。
                      
                   </p>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>For requirements related to color, refer to <a href="#distinguishable">Guideline 1.4</a>.
-                     </p>
+                     <p class="note-title marker">注記</p>
+                     <p>色に関する要件は、<a href="distinguishable.html">ガイドライン 1.4</a> を参照。
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Instructions are understandable by more people.</dd>
+                  <dt>目標</dt>
+                  <dd>より多くの人が説明を理解できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd> Describe controls by name, not just by appearance or location.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>コントロールを、外見又は位置ではなく、名前で記述する。</dd>
```

Comment:
「名前で記述する」というのは日本語としてやや違和感があります。ここの describe は「表現」「説明」「描写」くらいが良いように思います。

```suggestion
                  <dd>コントロールを、外見又は位置ではなく、名前で表現する。</dd>
```